### PR TITLE
Added my solution

### PR DESCRIPTION
**Pull request by BeyondInfinities**
This is a PR template. If you are adding a solution link to the curriculum, leave this as is. If not, delete it and write the message you wish.

Thank you,

The Odin Project team.

### CODE REVIEW

**Comment by 105ron** [comment]
You will need to create a repository for the file separate to the curriculum repo.

**Comment by BeyondInfinities** [comment]
The link was linked to my repository

**Comment by 105ron** [comment]
To do what you are requesting, you need to make a different branch to add this solution. In this PR you also included the script - it is advisable not to have you solutions in the curriculum repo as this becomes difficult to maintain and create PR's.

File: ruby_programming/basic_ruby_projects/caesar_cipher.md [hunk 1]
```diff
@@ -64,6 +64,7 @@ Submit a link below to this [file](https://github.com/TheOdinProject/curriculum/
 * [Andrew Huntington's Solution](https://github.com/AndrewHuntington/ruby_projects/tree/master/p1_caesar_cipher)
 * [Will's Solution](https://github.com/dunlapww/odin_project/blob/master/project_odin/caesar_cipher.rb)
 * [Nasser Abachi's Solution](https://github.com/abachi/theodinproject/tree/master/building-projects/caesar_cipher/)
+* [beyondinfinity's Solution](https://github.com/beyondinifnities/curriculum/ruby_programming/a.rb)
```
Comment (I3uckwheat) [inline ruby_programming/basic_ruby_projects/caesar_cipher.md:67]: This link is "not found"

**Pull request closed without merging**